apps/vor/pkg/school/school.go

```diff
@@ -83,7 +86,87 @@ func getClasses(server rest.Server, store postgres.SchoolStore) http.Handler {
 		return nil
 	})
 }
+func getClassSession(server rest.Server, store postgres.SchoolStore) http.Handler {
+	type responseBody struct {
+		Date string `json:"date"`
+	}
+	return server.NewHandler(func(w http.ResponseWriter, r *http.Request) *rest.Error {
+		classId := chi.URLParam(r, "classId")
+		attendance, err := store.GetClassSession(classId)
+		var response []responseBody
+
+		if len(attendance) > 0 {
+			attend := make(map[string]int)
```

Comment:
What does this variable do?

Reply:
The attendance table, only include those that are present. So we need a way to filter students from that class. That variables is to hold which student are present at the class for that session. And then we loop through the list of students of that class and check if the studentId is not in the 'attend' variables, it means the student is absent

Reply:
This should'nt be here. I'll make some quick fixes on the getClassSession function

Reply:
What is your thought about the logic to get class session? Is this the correct logic?

Reply:
I've been thinking about this. I think checking only from the attendance table probably isn't good enough.

Imagine this. There was a class on Monday, but the teacher forgot to do attendance and was asked about it on Tuesday. They would open the attendance page and found that there was no class on Monday since there is no data about that day on the attendance table yet right? They wouldn't be able to do the attendance for Monday.

One way I think would be better is by generating the sessions from the class data that we have.

ex. Given Class A, 9-10 am every Monday. We would generate 4th May (next session). 27th April (last session), 20th April, 13th April.... and so on up until the first attendance ever recorded for that class.

Reply:
So from my understanding, the add attendance endpoint needs to receive 'date' input. 
Lets say the endpoint was requested today, to get class A session and the attendance table only have attendance on 20th April. Whats the expected output?

Reply:
If the get session endpoint is called, it would return all sessions back to the 20th of April (the first session ever recorded).